src/ImageCommand.cc

```diff
@@ -63,6 +63,13 @@ void ImageCommand::enqueue_operations(VCL::Image& img, const Json::Value& ops)
                         get_value<int>(op, "width"),
                         get_value<int>(op, "height") ));
         }
+        else if (type == "flip") {
```

Comment:
This is getting long enough to make a switch case out of this but that will need some JSON level integer percolation instead of string. Anyway, something to think about as an optimization for us going forward as more operations get added. That's why don't make this a standard pull request way for adding ops. We need to think more.

Reply:
good idea, we can have a string table here.